Added documentation for GraphQL Client Errors (#7709)
getsentry/sentry-docs: src/platforms/dotnet/common/configuration/graphql-client-errors.mdx

    },
    new SystemTextJsonSerializer()
    );
```

```fsharp
let graphClient =
    new GraphQLHttpClient(
        fun options ->
            options.EndPoint <- Uri("http://your.app.server/graphql")
            options.HttpMessageHandler <- new SentryGraphQLHttpMessageHandler() // <-- Configure GraphQL use Sentry Message Handler
        ,
        new SystemTextJsonSerializer()
    )
```

By default, Sentry will capture GraphQL over HTTP client errors for requests to any GraphQL endpoint, but you can change this behavior by updating the `FailedRequestTargets` option with either regular expressions or plain strings. A plain string must contain at least one of the items from the list. Plain strings don't have to be full matches, meaning the URL of a request is matched when it contains a substring provided through the option:

```csharp
options.FailedRequestTargets.Add("foo");      // substring
options.FailedRequestTargets.Add("foo.*bar"); // regex
```

```fsharp
options.FailedRequestTargets.Add("foo")      // substring
options.FailedRequestTargets.Add("foo.*bar") // regex
```

> When the `SendDefaultPii` option is enabled, error events may contain PII data such as `Headers`, `Cookies` and the GraphQL request `query`. To scrub this data before it is sent, see [Scrubbing Sensitive Data](/platforms/dotnet/data-management/sensitive-data/).

Author: I think we just remove the bit about the list tbh - it's more confusing than helpful.

> These events are searchable and you can set alerts on them using things like the `http.url` and `http.status_code` properties. Learn more in our full [Searchable Properties](/product/sentry-basics/search/searchable-properties/) documentation.

Reviewer: ```suggestion
These events are searchable. You can set alerts for them using properties like the `http.url` and `http.status_code`. Read our [Searchable Properties](/product/sentry-basics/search/searchable-properties/) docs to learn more.
```

> ### Customize or Drop the Error Event
> 
> The captured error event can be customized or dropped in the `SetBeforeSend` callback:

Reviewer: ```suggestion
## Customize or Drop an Error Event

Any captured error event can be customized or dropped in the `SetBeforeSend` callback like this:
```
